dotcom-rendering/src/web/components/PinnedPost.importable.tsx

```diff
@@ -0,0 +1,191 @@
+		0deg,
+		${neutral[100]},
+		${neutral[100]} 40%,
+		rgba(255, 255, 255, 0)
+	);
+	height: 5rem;
+	position: absolute;
+	bottom: 0;
+	width: 100%;
+`;
+
+const button = css`
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	width: 20%;
+	background: ${news[300]};
+	cursor: pointer;
+	outline: none;
+	border: 0;
+	border-radius: 1rem;
+	height: 2rem;
+	bottom: -1rem;
+	margin-left: 0.625rem;
+	${textSans.small({ fontWeight: 'bold', lineHeight: 'tight' })};
+	color: ${neutral[100]};
+
+	&:focus div {
+		${focusHalo};
+	}
+	&:hover {
+		background: ${news[400]};
+	}
+	${from.tablet} {
+		margin-left: 3.75rem;
+	}
+	svg {
+		margin-right: 5px;
+		fill: ${neutral[100]};
+		width: 18px;
+		height: 18px;
+	}
+`;
+
+const expandedBody = css`
+	overflow: hidden;
+	height: auto;
+`;
+
+const collapsedBody = css`
+	max-height: 40vh;
+	overflow: hidden;
+`;
+
+export interface PinnedPostProps extends Props {
+	pinnedPost: Block;
+	format: ArticleFormat;
+	pageId: string;
+	webTitle: string;
+	adTargeting: AdTargeting;
+	host?: string;
+	ajaxUrl: string;
+	isLiveUpdate?: boolean;
+}
+
+export const PinnedPost = ({
+	pinnedPost,
+	format,
+	pageId,
+	webTitle,
+	adTargeting,
+	host,
+	ajaxUrl,
+	isLiveUpdate,
+}: PinnedPostProps): EmotionJSX.Element => {
+	const [expanded, setExpanded] = useState(false);
+	const [isBrowser, setIsBrowser] = useState(false);
+	const [showButton, setShowButton] = useState(false);
+
+	const collapse = () => setExpanded(false);
+	const expand = () => setExpanded(true);
+	const handleClick = () => (expanded ? collapse() : expand());
+
+	useEffect(() => {
+		if (
+			document.documentElement.scrollHeight >
+			document.documentElement.clientHeight
+		) {
+			setShowButton(true);
+		}
+	}, []);
+
+	useEffect(() => {
+		setIsBrowser(true);
+	}, []);
+
+	if (isBrowser) {
+		return (
+			<div css={pinnedPostContainer}>
+				<div css={pinnedPostRow}>
+					<PinIcon fill="white" />
+					{pinnedPost.blockFirstPublished && (
+						<time data-relativeformat="med" css={timeAgoText}>
+							From {timeAgo(pinnedPost.blockFirstPublished)}
+						</time>
+					)}
+				</div>
+
+				<div css={expanded ? expandedBody : collapsedBody}>
+					<LiveBlock
+						format={format}
+						block={pinnedPost}
+						pageId={pageId}
+						webTitle={webTitle}
+						adTargeting={adTargeting}
+						host={host}
+						ajaxUrl={ajaxUrl}
+						isLiveUpdate={isLiveUpdate}
+						isPinnedPost={true}
+					/>
+				</div>
+				{!expanded && <div css={overlay} />}
+				{showButton && (
+					<button
+						type="button"
+						aria-expanded={expanded}
+						onClick={handleClick}
+						css={button}
+					>
+						{expanded ? (
+							<>
+								<SvgMinus /> Show Less
+							</>
+						) : (
+							<>
+								<SvgPlus /> Show More
+							</>
+						)}
+					</button>
+				)}
+			</div>
+		);
+	}
+
+	// When a user doesn't have JS enabled we should use a pure css/html accordion instead
```

Comment:
Could we make this a `TODO` and link to the Trello ticket?

Reply:
Done 👍